spec/features/user_logout_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+feature 'User logs out' do
+  scenario 'destroy website session' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.